app/static/js/custom-dropdown.js

```diff
@@ -0,0 +1,37 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const allSections = document.querySelectorAll("section")
+  const sectionIds = Array.apply(null, allSections).map((section) =>
+    section.getAttribute("id"),
+  )
+  const sectionMeta = sectionIds
+    .map((id) => {
+      return { id, toggle: document.getElementById(id + "_toggle") }
+    })
+    .filter((meta) => meta.toggle)
+    .map((meta) => {
+      return {
+        id: meta.id,
+        toggle: meta.toggle,
+        charts: {
+          ber: document.getElementById(meta.id + "_ber"),
+          ger: document.getElementById(meta.id + "_ger"),
+        },
+      }
+    })
+
+  sectionMeta.forEach((section) => {
+    section.toggle.addEventListener("change", () => {
+      const locations = ["ber", "ger"]
+      const selectedChartId = section.toggle.value
+      const activeChartLocation = selectedChartId.slice(
+        -3,
+        selectedChartId.length,
+      )
```

Comment:
This `-3` is a [magic number](https://www.wikiwand.com/en/Magic_number_(programming)). :D 

I can see where it’s coming from within this context, but this might be a bit risky to keep around like that. Having it in a separate variable would already help, I think. And if you also skip the second parameter of `slice()`, this becomes a bit clearer:

```suggestion
      const SUFFIX_LENGTH = "ber".length
      const selectedChartId = section.toggle.value
      const activeChartLocation = selectedChartId.slice(-SUFFIX_LENGTH)
```

Reply:
Haha: "This has been referred to as breaking one of the oldest rules of programming, dating back to the [COBOL](https://www.wikiwand.com/en/COBOL), [FORTRAN](https://www.wikiwand.com/en/FORTRAN) and [PL/1](https://www.wikiwand.com/en/PL/1) manuals of the 1960s." Nice! Fixed it. also like it way better that way🤗